grpc/src/grpc.rs

```diff
@@ -90,6 +94,65 @@ where
 
     // cosmos.bank
 
+    /// Get balance of coins with [`BOND_DENOM`] for the given address, together with a proof,
+    /// and verify the returned balance against the corresponding block's [`AppHash`].
+    ///
+    /// NOTE: the balance returned is the balance reported by the parent block of
+    /// the provided header. This is due to the fact that for block N, the block's
+    /// [`AppHash`] is the result of applying the previous block's transaction list.
+    ///
+    /// [`AppHash`]: tendermint::hash::AppHash
+    pub async fn get_verified_balance(
```

Comment:
should this be renamed to `balance_for_account` to match celestia grpc client name?

Reply:
yes